packages/core/src/request.ts

```diff
@@ -236,8 +239,11 @@ export class Request<UserData extends Dictionary = Dictionary> {
 
     /** Tells the crawler processing this request to skip the navigation and process the request directly. */
     set skipNavigation(value: boolean) {
-        if (!this.userData.__crawlee) (this.userData as Dictionary).__crawlee = { skipNavigation: value };
-        else this.userData.__crawlee.skipNavigation = value;
+        if (!this.userData.__crawlee) {
+            (this.userData as Dictionary).__crawlee = { skipNavigation: value };
```

Comment:
Not a blocker but I am surprised you need this cast here, but not in the else case

Reply:
It is needed, don't ask me why :P 

Otherwise you get: `TS2339: Property '__crawlee' does not exist on type 'UserData'`.